### ISSUE TITLE
Не работает плагин De-Esser

### ISSUE THREAD

**Issue by Desynci** [asklimanov/fifine-k658#1]
Привет! Архив `accusonus4` пропал из файлов в ВК. 
Я скачал его на другом сайте — APO его видит и даже способна открыть панельку, однако при его включении звук с микрофона пропадает. 

Windows распознаёт уровень звука, но на Audacity и «прослушивать с этого устройства» лишь тишина. APO также показывает неверные `-inf peak` и `0.0 latency`.
Достаточно отключить плагин, и звук появляется вновь, а показатели возвращаются в норму.

Я полагаю, что мне попался кривой кряк. Если это так, подскажете, где скачать нормальную версию?

**Comment by agragregra** [comment]
Файл не пропал, вы попробуйте вбивать по буквам accusonus4, а не вставлять скопированный текст.
Если со сторонних - качайте accusonus 4

**Comment by agragregra** [comment]
Можете скачать в принципе, любой accusonus и установить 10 в качестве значения, разницы не будет.
<img width="153" alt="Screenshot_1" src="https://github.com/asklimanov/fifine-k658/assets/10989424/89c4afb8-92ae-4358-8fa0-ed698c2eb541">

**Comment by Desynci** [comment]
> Файл не пропал, вы попробуйте вбивать по буквам accusonus4, а не вставлять скопированный текст. Если со сторонних - качайте accusonus 4

Да, это действительно помогло: файл высветился. Закинул его — теперь всё работает. 
Спасибо!

**Comment by asklimanov** [comment]
Пусть висит открытым.
